src/components/shared/TokenAMMPriceBadge.tsx

```diff
@@ -0,0 +1,88 @@
+import { LoadingOutlined } from '@ant-design/icons'
+import { useContext } from 'react'
+import { ThemeContext } from 'contexts/themeContext'
+import { useUniswapPriceQuery } from 'hooks/ERC20UniswapPrice'
+import { formattedNum } from 'utils/formatNumber'
+import UniswapLogo from 'components/icons/Uniswap'
+import { Tooltip } from 'antd'
+import { t } from '@lingui/macro'
+
+type exchangeName = 'Uniswap'
```

Comment:
types are TitleCase

Reply:
ahh 🙈 . Thanks for catching it. See https://github.com/jbx-protocol/juice-interface/pull/323 😘

Reply:
https://github.com/jbx-protocol/juice-interface/pull/311/commits/32821cbcbc0f50a9eb0c8fe14740b46185b00df0